oca_dependencies.txt

```diff
@@ -12,3 +12,4 @@ purchase-workflow
 sale-workflow
 delivery-carrier
 web
+product-atribute
```

Comment:
faltou o repo brand https://github.com/OCA/brand onde se encontra a nova dependencia product_brand. As ferramentas de CI instalam agora tudo pelo pip e por isso a CI passa, mas se eh para ser consistente eh bom adicionar o repo aqui para ajudar que for pegar no projeto.